### PR TITLE
Release Blog for Collector v0.34.0

### PR DESCRIPTION
**Pull request by vasireddy99**
This PR is created to add release blog for v0.34.0

### CODE REVIEW

File: src/content/BlogPosts/blogPosts.yaml [hunk 1]
```diff
@@ -6,6 +6,14 @@ description:
 path: /blog
 
 blogs:
+  - title: "AWS Distro for OpenTelemetry v0.34.0 is now available"
+    author: "Pavan Sai Vasireddy"
+    date: "16-October-2023"
+    body:
+      "AWS Distro for OpenTelemetry v0.34.0 is now available available. You can download the latest AWS Distro for OpenTelemetry Collector image
```
Comment (mhausenblas) [inline src/content/BlogPosts/blogPosts.yaml:13]: I know we're excited that it's available but one available is sufficient ;)

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+**Upstream changelog**
+
+* [OpenTelemetry Collector v0.85.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.85.0)
+* [OpenTelemetry Collector v0.86.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.86.0)
+* [OpenTelemetry Collector Contrib v0.85.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.85.0)
+* [OpenTelemetry Collector Contrib v0.86.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.86.0)
+
+**New Components**
+
+The following components are now included in the ADOT Collector behind the feature gate.
```
Comment (mhausenblas) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx:26]: ADOT Collector behind the feature gate ==> ADOT Collector, behind a feature gate

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+To enable the feature flag in, you can pass the `--feature-gates=+adot.receiver.filelog,+adot.exporter.awscloudwatchlogs,+adot.extension.file_storage` in the command line parameter.
```
Comment (mhausenblas) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx:27]: To enable the functionality behind the feature gate, pass the …

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+[FileLog Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver) -
+See the [FileLog Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver#filelog-receiver)documentation page to for the usage.
```
Comment (mhausenblas) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx:30]: insert space between link and "documentation" and drop "page"

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+[AWS CloudWatch Logs Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/awscloudwatchlogsexporter) -
+Please refer to [CloudWatch Logs Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/awscloudwatchlogsexporter#aws-cloudwatch-logs-exporter) documentation page for usage
```
Comment (mhausenblas) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx:33]: same here, drop "page"

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+[File Storage Extension](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/storage/filestorage) -
+See the [storage extension](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/storage/filestorage#file-storage) documentation page to see how it can be used in your
```
Comment (mhausenblas) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx:36]: same here, drop "page"

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+configurations.
+
+** Important: ADOT Collector v0.34.0 Breaking Change **
+
+Users of the statsd receiver, please refer to GitHub Issue - [Warning: StatsD Receiver → EMF Exporter
```
Comment (mhausenblas) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx:41]: `statsd`

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+configurations.
+
+** Important: ADOT Collector v0.34.0 Breaking Change **
+
+Users of the statsd receiver, please refer to GitHub Issue - [Warning: StatsD Receiver → EMF Exporter 
+Metric Pipeline Breaking Change](https://github.com/aws-observability/aws-otel-collector/issues/2249) for information on an upcoming breaking change.
+
+** Important: ADOT Collector v0.35.0 Breaking Change
+
+Users of the awscontainerinsightreceiver, please refer to the [GitHub Issue - Warning: Container Image Default User Change → Important
```
Comment (mhausenblas) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx:46]: `awscontainerinsightreceiver`

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+consideration for AWSContainerInsightReceiver](https://github.com/aws-observability/aws-otel-collector/issues/2317) for more information on an upcoming breaking change.
+
+- 
+
+Detailed release notes are on [GitHub](https://github.com/aws-observability/aws-otel-collector/releases).
```
Comment (mhausenblas) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx:51]: "Detailed release notes are on" --> "Detailed release notes are available via"

**Review by mhausenblas — CHANGES_REQUESTED** [review]
Thanks and find my feedback inline

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx (hunk 2 above)
Comment (vasireddy99) [reply src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx:26]: Fixed

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx (hunk 4 above)
Comment (vasireddy99) [reply src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx:30]: fixed

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx (hunk 5 above)
Comment (vasireddy99) [reply src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx:33]: fixed

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx (hunk 7 above)
Comment (vasireddy99) [reply src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx:41]: fixed

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx (hunk 8 above)
Comment (vasireddy99) [reply src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx:46]: fixed

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx (hunk 9 above)
Comment (vasireddy99) [reply src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx:51]: fixed

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+**Upstream changelog**
+
+* [OpenTelemetry Collector v0.85.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.85.0)
+* [OpenTelemetry Collector v0.86.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.86.0)
+* [OpenTelemetry Collector Contrib v0.85.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.85.0)
+* [OpenTelemetry Collector Contrib v0.86.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.86.0)
+
+**New Components**
+
+The following components are now included in the ADOT Collector, behind a feature gate.
```
Comment (bryan-aguilar) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.34.0.mdx:26]:
```suggestion
The following components are now included in the ADOT Collector behind a feature gate.
```

**Pull request merged**